src/components/ratings.tsx

```diff
@@ -9,7 +9,9 @@ export interface RatingsProps {
     Icon: React.StatelessComponent<IconWrapperProps>;
```

Comment:
this type needs to be extended to add `borderColor`, as I currently get an error